src/modules/analysis/AnalysisAddComponent.js

```diff
@@ -142,6 +143,42 @@ const AnalysisAdd = (
     <PageHeader style={{ paddingLeft: 20 }}>
       {I18n.t('musit.analysis.registeringAnalysis')}
     </PageHeader>
+    <Form horizontal style={{ paddingLeft: 10 }}>
+      <FormGroup>
+        <Label label="Type analyse" md={1} />
+        <Col md={2}>
+          <select
```

Comment:
Use one way to setup the select element. Here's an example from the react-bootstrap project:
https://github.com/react-bootstrap/react-bootstrap/blob/master/docs/examples/FormControls.js#L78